message_ix/model/MESSAGE/sets_maps_def.gms

```diff
@@ -243,6 +246,7 @@ Sets
     type_emission                           types of emission aggregations
     cat_emission(type_emission,emission)    mapping of emissions to respective categories
     type_tec_land(type_tec)                 dynamic set whether emissions from land use are included in type_tec
+    full_balance(commodity)                 subset of 'commodity' to mark all commodities in the full commodity balance
```

Comment:
just for clarity, I would suggest the name `commodity_balance_eq` here instead of `full` just because it's a bit more exact

Reply:
or maybe commodity_balance_lt
and stick with '_gt' and '_lt' for the two commodity balances all over the model, also concerning the equation/slack variable names

Reply:
The tricky thing here is that if a commodity belongs to this set it will be placed in *both* gt and lt at the moment. Perhaps it is best to get @danielhuppmann's thoughts on this.

Reply:
agree with @gidden, whatever commodity (or probably better commodity-and-level?) is specified here must balance with equality.